core/client/ui/user-onboarding.js

```diff
@@ -0,0 +1,110 @@
+const ONBOARDING_LAST_STEP = 3;
+const keyboards = ['q', 'z', 's', 'd', 'up', 'down', 'right', 'left'];
+const isMobile = window.innerWidth < 480;
```

Comment:
Maybe we could move the logic of isMobile into `helpers.js` as there is already a function to check the platform in `scene-world.js` : 
`window.matchMedia('(pointer: coarse)').matches`

In both cases I have the impression that this is not the optimal way to check the platform btw